main.tf

```diff
@@ -1,57 +1,161 @@
 locals {
-  security_group_enabled = module.this.enabled && var.security_group_enabled
-  is_external            = module.this.enabled && var.security_group_enabled == false
-  use_name               = var.use_name_prefix ? null : module.this.id
-  use_name_prefix        = var.use_name_prefix ? format("%s%s", module.this.id, module.this.delimiter) : null
-  id                     = local.is_external ? join("", data.aws_security_group.external.*.id) : join("", aws_security_group.default.*.id)
-  arn                    = local.is_external ? join("", data.aws_security_group.external.*.arn) : join("", aws_security_group.default.*.arn)
-  name                   = local.is_external ? join("", data.aws_security_group.external.*.name) : join("", aws_security_group.default.*.name)
-  rules = module.this.enabled && var.rules != null ? {
-    for indx, rule in flatten(var.rules) :
-    format("%v-%v-%v-%v-%s",
-      rule.type,
-      rule.protocol,
-      rule.from_port,
-      rule.to_port,
-      try(rule["description"], null) == null ? md5(format("Managed by Terraform #%d", indx)) : md5(rule.description)
-    ) => rule
-  } : {}
-}
+  enabled = module.this.enabled
+  inline  = var.inline_rules_enabled
+
+  allow_all_egress = local.enabled && var.allow_all_egress
+
+  default_rule_description = "Managed by Terraform"
+
+  # Because Terraform formatting for `not` (!) changes between versions 0.13 and 0.14, use == false instead
```

Comment:
nit: Since we're going with 0.14, we can probably remove this comment.

```suggestion
```

Reply:
We are not going with 0.14 yet. We are not _supporting_ 0.13 in the sense that we are not fixing problems that arise when using 0.13 that are fixed in 0.14, but we are allowing people to use 0.13 and it mostly works except for when you use derived values in some places.